app/concerns/fetchable.rb

```diff
@@ -47,6 +47,7 @@ module Fetchable
     scope :unfetched, -> { where('ISNULL(last_crawled_at)') }
 
     before_validation :normalize_url
+    before_validation :truncate_error_message
```

Comment:
The `last_crawl_status` field will not always be an error message. (When things go right, its value is `'OK'`.) So I would rename this method to `truncate_last_crawl_status`.